Cargo.toml

```diff
@@ -185,5 +185,5 @@ alloy-json-abi = { git = "https://github.com/alloy-rs/core", rev = "7cb5bb220dfe
 alloy-primitives = { git = "https://github.com/alloy-rs/core", rev = "7cb5bb220dfeacb780c55f4ca9f49650ae0bcce6" }
 alloy-sol-macro = { git = "https://github.com/alloy-rs/core", rev = "7cb5bb220dfeacb780c55f4ca9f49650ae0bcce6" }
 alloy-sol-type-parser = { git = "https://github.com/alloy-rs/core", rev = "7cb5bb220dfeacb780c55f4ca9f49650ae0bcce6" }
-alloy-sol-types = { git = "https://github.com/alloy-rs/core", rev = "7cb5bb220dfeacb780c55f4ca9f49650ae0bcce6" }
+alloy-sol-types = { git = "https://github.com/alloy-rs/core", rev = "e79fe576a2aea713370e9d96297ec5a458963c39" }
```

Comment:
these should not diverge, so should bump all revs

Reply:
ok thanks, I change them all now